Library.Console/Program.cs

```diff
@@ -1,188 +1,203 @@
 ﻿using Library.Console;
 
-bool isContinue = true;
-IBooksManager booksManager;
+public class Program
+{
+    private static IBooksManager booksManager;
+    private static List<Book> books = new List<Book>();
 
-Console.WriteLine("How do you want to save the file?");
-Console.WriteLine("1. Json");
-Console.WriteLine("2. Xml");
+    private static bool isContinue = true;
 
-int res = int.Parse(Console.ReadLine());
-switch (res)
-{
-    case 1:
-        booksManager = new BooksJsonManager();
-        break;
-    case 2:
-        booksManager = new BooksXmlManager();
-        break;
-    default:
-        booksManager = new BooksJsonManager();
-        break; 
-}
-List<Book> books = booksManager.Load();
-
-while (isContinue)
-{
-    Console.WriteLine("1. Add Book");
-    Console.WriteLine("2. Show List of Books");
-    Console.WriteLine("3. Update List of Books");
-    Console.WriteLine("4. Delete Book");
-    Console.WriteLine("5. Exit");
+    public static void Main(string[] args)
+    {
+        Console.WriteLine("How do you want to save the file?");
+        Console.WriteLine("1. Json");
+        Console.WriteLine("2. Xml");
 
-    Console.WriteLine();
+        int res = int.Parse(Console.ReadLine() ?? string.Empty);
+        switch (res)
+        {
+            case 1:
+                booksManager = new BooksJsonManager();
+                break;
+            case 2:
+                booksManager = new BooksXmlManager();
+                break;
+            default:
+                booksManager = new BooksJsonManager();
+                break;
+        }
```

Comment:
You can use new C# syntax for switch like this:
```suggestion
booksManager = res switch
        {
            1 => new BooksJsonManager(),
            2 => new BooksXmlManager(),
            _ => new BooksJsonManager()
        };
        ```

Reply:
oke